R/robustbayesianmetaanalysis.R

```diff
@@ -551,39 +548,39 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
 .robmaReady               <- function(options) {
```

Comment:
Perhaps use a verb as an identifier. I suspect that this function checks whether the analysis is ready. If so, then something such as '.robmaCheckReady' might be informative.

Reply:
changed